Pagination/MongoTrait/FilterTypeStrategy/Strategies/BooleanFilterStrategy.php

```diff
@@ -28,10 +28,13 @@ public function support($type)
      */
     public function generateFilter($name, $value, $documentName)
```

Comment:
I cannot because I should have the same signature that in the `FilterTypeInterface`